packages/flutter_driver/lib/src/driver/vmservice_driver.dart

```diff
@@ -130,7 +130,7 @@ class VMServiceFlutterDriver extends FlutterDriver {
     // Attempts to resume the isolate, but does not crash if it fails because
     // the isolate is already resumed. There could be a race with other tools,
     // such as a debugger, any of which could have resumed the isolate.
-    Future<dynamic> resumeLeniently() {
+    Future<dynamic> resumeLeniently(VMIsolate isolate) {
```

Comment:
nit: prefer not shadowing parameter names in the enclosing class, which can get confusing.  I don't have a great idea for a name here.

This does make me wonder if there are other cases we should be explicitly passing the isolate reference around though.